user/enterprise.md

```diff
@@ -27,6 +27,9 @@ Enterprise-specific features include:
 installation as the authentication and authorization layer for Travis CI
 Enterprise, allowing you to leverage your existing LDAP or SAML with no
 extra configuration.
+* **Support for multiple Version Control Systems:** Starting from Travis CI 
+Enterprise 3.x, you can use also different source code management solutions
```

Comment:
"you can use also different" change to "you can also use different"

Reply:
applied in subsequent commit